ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -843,8 +845,8 @@
                                                      (or (nil? change-date)
                                                          (date-in-the-past? (.toLocalDate change-date))))
                                                    (sort-by :gtfs/change-date route-change-infos)))
-          ;; Set change date to future (every 4 weeks at monday) - This is the day when changes are detected for next time
-          new-change-date (time/sql-date (time/native->date (.plusDays (time/beginning-of-week (.toLocalDate (time/now))) 28)))
+          ;; Set change date to future (every 2 weeks at monday) - This is the day when changes are detected for next time
```

Comment:
Minor style issue: IMHO constant should define the interval and comments should not explicitly refer to the interval length (2 weeks) anymore. Reason: later someone changes the constant value and is only likely not to update the comments where the symbol was referred to ==> comments will be out of date :)